File: sammel_server/src/main/java/database/termine/TerminDetails.kt
package database.termine

import javax.persistence.*
import javax.persistence.CascadeType.ALL

@Entity
@Table(name = "TerminDetails")
class TerminDetails {

    @Id
    @Column(name = "termin_id")
    var termin_id: Long? = null

    @OneToOne(cascade = [ALL])
    @MapsId
    @JoinColumn(name = "termin_id")
    var termin: Termin? = null

    @Column
    var treffpunkt: String? = null

    @Column
    var beschreibung: String? = null

    @Column
    var kontakt: String? = null

    constructor()

    constructor(termin_id: Long?, treffpunkt: String?, beschreibung: String?, kontakt: String?) {
        this.termin_id = termin_id
        this.treffpunkt = treffpunkt
        this.beschreibung = beschreibung
        this.kontakt = kontakt
    ***REMOVED***
***REMOVED***

File: sammel_app/lib/model/TerminDetails.dart
class TerminDetails {
  int id;
  String treffpunkt;
  String beschreibung;
  String kontakt;

  TerminDetails(this.treffpunkt, this.beschreibung, this.kontakt);

  TerminDetails.fromJSON(Map<String, dynamic> json)
      : id = json['id'],
        treffpunkt = json['treffpunkt'],
        beschreibung = json['beschreibung'],
        kontakt = json['kontakt'];

  Map<String, dynamic> toJson() => {
        'id': id,
        'treffpunkt': treffpunkt,
        'beschreibung': beschreibung,
        'kontakt': kontakt
      };
}
